src/OpenFeature.Contrib.Providers.Flagd/Resolver/InProcess/CustomEvaluators/FlagdProperties.cs

```diff
@@ -17,15 +17,15 @@ internal class FlagdProperties
         internal FlagdProperties(object from)
         {
             //object value;
-            if (from is Dictionary<string, object> dict)
+            if (from is IDictionary<string, object> dict)
```

Comment:
This stumped me for a moment. `from` here implements `IDictionary`, but is not actually an instance of `Dictionary`, so this evaluated to false, resulting in no injected props. This fixes it!